.github/workflows/rfs_pr_e2e_test.yml

```diff
@@ -18,23 +18,13 @@ jobs:
   rfs-e2e-aws-test:
     runs-on: ubuntu-latest
     steps:
-      - name: Determine Github repository and branch
-        id: determine-repo-vars
-        run: |
-          if [[ "${GITHUB_EVENT_NAME}" == "pull_request_target" ]]; then
-            branch_name="${{ github.event.pull_request.head.ref }}"
-            pr_repo_url="https://github.com/${{ github.event.pull_request.head.repo.full_name }}.git"
-          else
-            branch_name="${{ github.ref_name }}"
-            pr_repo_url="https://github.com/${{ github.repository }}.git"
-          fi
-          echo "Running jenkins test on repo: $pr_repo_url and branch: $branch_name"
-          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
-          echo "pr_repo_url=$pr_repo_url" >> $GITHUB_OUTPUT
+      - env:
+          BRANCH_NAME: ${{ github.event.pull_request.head.ref || github.ref_name }}
+          REPO_BASE: ${{ github.event.pull_request.head.repo.full_name || github.repository }}
```

Comment:
While injection is less likely, lets move these to another step and run `tr -cd '[:alnum:]_-./'` on the both parameters to limit what can be passed into jenkins job_params.

Reply:
Added now